.github/workflows/publish-docker.yaml

```diff
@@ -48,11 +48,13 @@ jobs:
             echo "DOCKER_USERNAME=${{ secrets.DOCKERHUB_USER }}" >> $GITHUB_ENV
             echo "DOCKER_PASSWORD=${{ secrets.DOCKERHUB_TOKEN }}" >> $GITHUB_ENV
             echo "HUB=apache" >> $GITHUB_ENV
+            echo "DOCKER_TAG=${{ github.ref_name }}" >> $GITHUB_ENV
```

Comment:
Should be `github.event.release.tag_name`

Reply:
👍  I find it in https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#get-a-release